ivy/functional/backends/paddle/experimental/elementwise.py

```diff
@@ -245,7 +251,7 @@ def fix(
     *,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    paddle.floor(x, name="fix")
```

Comment:
AssertionError: The length of results from backend paddle and ground truth framework tensorflow does not match